source/codegen/metadata/nidmm/functions.py

```diff
@@ -1624,7 +1624,7 @@
     },
     "InitExtCal": {
         "init_method" : True,
-        "codegen_method" : None,
+        "custom_close" : "CloseExtCal(id,NIDMM_EXTCAL_ACTION_ABORT)",
```

Comment:
[Minor] Can you add a space between the comma and second argument?

Reply:
Done